src/components/App.js

```diff
@@ -1,104 +1,90 @@
+    }
+  };
 
-        //  const isValid = this.validate();
-        // if (isValid) {
-        //     console.log(this.state);
-        // }
-    };
+  handleSubmit = (event) => {
+    event.preventDefault();
+    const arrayVar = this.state.itemsList;
+    arrayVar.push(
+      <GradientItem
+        inputValueFirst={this.state.inputValueFirst}
+        inputValueSecond={this.state.inputValueSecond}
+        deleteItem={this.deleteItem}
+      />
+    );
+    this.setState({
+      isSubmitted: true,
+      itemsList: arrayVar,
+    });
+  };
 
-    render() {
-        console.log('itemsArray', this.state.itemsList);
-        //console.log('changed', this, this.state, this.state.inputValueFirst, this.state.inputValueSecond);
-        const items = this.state.itemsList.map((item, index) => <li key={index}>{item}</li>);
-        return (
-            <div className="App">
-                <form onSubmit={this.handleSubmit}>
-                    <label><input placeholder="hex1"
-                                  name="inputValueFirst"
-                                  value={this.state.inputValueFirst}
-                                  type="text"
-                                  onChange={this.handleChange}
-                                  required/>
-                    </label>
-                    <label><input placeholder="hex2"
-                                  name="inputValueSecond"
-                                  value={this.state.inputValueSecond}
-                                  type="text"
-                                  onChange={this.handleChange}
-                                  required/>
-                    </label>
-                    <button disabled={this.state.isDisabled} type="submit">Add Gradient</button>
-                </form>
-                <ul style = {{listStyle: 'none'}}>
-                    {items}
-                </ul>
-            </div>
-        )
-    }
+  render() {
+    console.log("itemsArray", this.state.itemsList);
+    const items = this.state.itemsList.map((item, index) => (
+      <li key={index}>{item}</li>
+    ));
+    return (
+      <div className="App">
+        <form onSubmit={this.handleSubmit}>
+          <label>
+            <input
+              placeholder="hex1"
+              name="inputValueFirst"
+              value={this.state.inputValueFirst}
+              type="text"
+              onChange={this.handleChange}
+              required
+            />
+          </label>
+          <label>
+            <input
+              placeholder="hex2"
+              name="inputValueSecond"
+              value={this.state.inputValueSecond}
+              type="text"
+              onChange={this.handleChange}
+              required
+            />
+          </label>
+          <button disabled={this.state.isDisabled} type="submit">
+            Add Gradient
+          </button>
+        </form>
+        <ul style={{ listStyle: "none" }}>{items}</ul>
```

Comment:
Стили выноси в `CSS`.
```javascript
const { colorSets } = this.state;
```
```suggestion
        <ul style={{ listStyle: "none" }}>
          {colorSets.map(colorSet => (
            <GradientItem key={colorSet.id} colorSet={colorSet} />
          ))}
        </ul>
```